assets/src/dashboard/components/pageStructure/navigationComponents.js

```diff
@@ -83,16 +87,4 @@ export const AppInfo = styled.div`
   color: ${({ theme }) => theme.colors.gray500};
 `;
 
-export const WebStoriesHeading = styled.h1`
-  width: 100%;
-  margin-bottom: 0;
-  font-family: ${({ theme }) => theme.typography.family.secondary};
```

Comment:
This means we are no longer using Roboto anywhere on the dashboard. Can we remove `secondary` from the `theme.js`? Then we can also stop loading the Roboto font in general on this screen (yay for performance!)

Reply:
good spotting, yup! i'll remove it and also from the includes/Dashboard.php